src/services/eventHandlers/InteractionCoordinator.ts

```
@@ -0,0 +1,761 @@
+    this.state.measurementSystem = measurementSystem;
+  }
+
+  /** Set the wall culling reference */
+  public setWallCulling(wallCulling: SimpleWallCulling): void {
+    this.state.wallCulling = wallCulling;
+  }
+
+  /** Set the event bus for decoupled communication */
+  public setEventBus(eventBus: SceneEventBus): void {
+    // Clean up previous subscriptions
+    this.state.eventBusUnsubscribers.forEach(unsub => unsub());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)